src/Components/Web/src/Forms/Mapping/FormValueMappingContext.cs

```diff
@@ -13,24 +13,31 @@ public class FormValueMappingContext
     /// <summary>
     /// Initializes a new instance of <see cref="FormValueMappingContext"/>.
     /// </summary>
-    /// <param name="formName">The name of the form to map data from.</param>
+    /// <param name="mappingScopeName">The name of the current <see cref="FormMappingScope"/>. Values will only be mapped if the incoming data corresponds to this scope name.</param>
+    /// <param name="restrictToFormName">If set, indicates that the mapping should only receive values if the incoming form matches this name. If null, the mapping should receive data from any form in the mapping scope.</param>
     /// <param name="valueType">The <see cref="Type"/> of the value to map.</param>
     /// <param name="parameterName">The name of the parameter to map data to.</param>
-    public FormValueMappingContext(string formName, Type valueType, string parameterName)
+    public FormValueMappingContext(string mappingScopeName, string? restrictToFormName, Type valueType, string parameterName)
```

Comment:
Second parameter "sounds like a boolean" but it's not. why not just "formName"?

Reply:
> Reviewing it now I see that's inconsistent with the name MappingScopeName so perhaps I should rename these two to AllowMappingScopeName and AllowFormName.

Making this change in my follow-up PR.